core/include/kernel/msg_param.h

```diff
@@ -45,6 +45,19 @@ enum msg_param_mem_dir {
 };
 
 /**
+ * msg_param_mobj_from_noncontig_param() - construct mobj from non-contigous
+ * list of pages.
+ *
+ * @param - pointer to msg_param with OPTEE_MSG_ATTR_NONCONTIG flag set
+ * @bool  - true if buffer needs to be mapped into OP-TEE address space
+ *
+ * return:
+ *	mobj or NULL on error
+ */
+struct mobj *msg_param_mobj_from_noncontig_param(
```

Comment:
Just drop the last "_param" at the end, it kind of redundant any way. So my suggestion is `msg_param_mobj_from_noncontig()`.

Important stuff both of them. :-)